Detectors/MUON/MCH/Simulation/src/Response.cxx

```diff
@@ -21,6 +21,40 @@
 
 using namespace o2::mch;
 
+//_____________________________________________________________________
+void Response::init()
```

Comment:
as mentioned above, don't see the necessity of that init method. Put that code in the constructor where it belongs